workflow_tests/jest.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
     verbose: true,
     transform: {
-        '^.+\\.jsx?$': 'babel-jest',
+        '^.+\\.tsx?$': 'ts-jest',
```

Comment:
Shouldn't we keep the previous transform for JS files too?

Reply:
Updated to `'^.+\\.(js|jsx|ts|tsx)$': ['ts-jest', {babelConfig: true}],`